client/src/components/pages/Home/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Icon from 'antd/lib/icon';
```

Comment:
What is the different between those
```suggestion
import { Icon } from 'antd';
```

Reply:
they are the same , I use yours in the file.
thanks